### DISCUSSION TITLE
problem with chatwoot api webhook.

### DISCUSSION THREAD

**Discussion by ULYBR** [chatwoot/chatwoot#7970] [Chatwoot APIs]
I'm facing a problem with the api, every time I create a contact through the api the webhook is not triggering an event even though the webhook is registered on the platform with this event. When I do it via the web, it triggers the create contact event.

If anyone knows how I can get around this problem, I would be very grateful to get help in this case, I have already read the documentation and researched it but I believe there is a problem with the webhook in this case.

**Comment by sojan-official** [comment]
@ULYBR This shouldn't be the case. there shouldn't be any differences when the contact is created via web / API. can you share the curl request that you use to create contact ?

**Reply by ULYBR** [reply]
Because I thought it was something about me not knowing how to use the API, but I'm creating contact with the API token and passing the required fields. Even so, it only triggers the webhook when I create the contact in the app version of the account site.
{
  "meta": {
    "instanceId": "599ae3c347e573b42271ec087dfbb940187223c81ca656af8c21c894af853ce7"
  },
  "nodes": [
    {
      "parameters": {
        "requestMethod": "POST",
        "url": "https://app.chatwoot.com/api/v1/accounts/77259/conversations",
        "allowUnauthorizedCerts": true,
        "responseFormat": "string",
        "options": {},
        "bodyParametersUi": {
          "parameter": [
            {
              "name": "source_id"
            },
            {
              "name": "inbox_id",
              "value": "32893"
            },
            {
              "name": "contact_id",
              "value": "={{$node[\"Create contact chatwoot\"].json[\"data\"][\"payload\"][\"contact\"][\"id\"]}}"
            },
            {
              "name": "status",
              "value": "pending"
            }
          ]
        },
        "headerParametersUi": {
          "parameter": [
            {
              "name": "api_access_token",
              "value": "Token Api"
            }
          ]
        }
      },
      "id": "0d826153-9834-4ad7-8d08-734d61d52b92",
      "name": "Create conversa chatwoot1",
      "type": "n8n-nodes-base.httpRequest",
      "typeVersion": 2,
      "position": [
        -520,
        2900
      ]
    }
  ],
  "connections": {}
}
